web/gds-user-ui/src/locales/de/messages.po

```diff
@@ -203,7 +203,7 @@ msgid "Already have an account?"
 msgstr ""
 
 #: src/components/NetworkAnnouncements/index.tsx:21
-msgid "An error has occurred to load annoucements"
+msgid "An error has occurred to load announcements"
```

Comment:
@elysee15 @masskoder  this appears to be a typo in the app - can you please fix?